### PR TITLE
Default `icon` according to `color` variant in the `Alert`

### PR DESCRIPTION
**Pull request by dlouhak**

### CODE REVIEW

File: packages/web-react/src/components/Alert/README.md [hunk 1]
```diff
@@ -15,15 +15,29 @@ import { Alert } from '@lmc-eu/spirit-web-react/components';
 <Alert color="informative" isCentered>Hey! Pay attention!</Alert>
 ```
 
+## Default icons according to color variant
```
Comment (adamkudrna) [inline packages/web-react/src/components/Alert/README.md:18]: Title Case Please 🙏🏻

File: packages/web-react/src/components/Alert/README.md [hunk 2]
```diff
@@ -15,15 +15,29 @@ import { Alert } from '@lmc-eu/spirit-web-react/components';
...
-| `iconName`     | `string`                                     | Based on `color` | no       | Icon used in Alert         |
-| `isCentered` . | `boolean`                                    | `false`          | no       | If true, Alert is centered |
+| Prop name      | Type                                         | Default   | Required | Description                |
+| -------------- | -------------------------------------------- | --------- | -------- | -------------------------- |
+| `children`     | `ReactNode`                                  | -         | yes      | Content of the Alert       |
+| `color`        | [Emotion Color dictionary][dictionary-color] | `success` | no       | Color of the component     |
+| `iconName`     | `string`                                     | `info` \* | no       | Icon used in Alert         |
+| `isCentered` . | `boolean`                                    | `false`   | no       | If true, Alert is centered |
+
+(\*) For every emotion color is defined default icon from the [Icon package] or your source of icons.
```
Comment (adamkudrna) [inline packages/web-react/src/components/Alert/README.md:37]: 🇨🇿 👀 

```suggestion
(\*) For each emotion color, a default icon is defined. The icons come from the [Icon package], or from your custom source of icons.
```

AI will surely provide even better phrasing.

**Review by adamkudrna — COMMENTED** [review]
Cool! 👍🏻

File: packages/web-react/src/components/Alert/README.md (hunk 2 above)
Comment (dlouhak) [reply packages/web-react/src/components/Alert/README.md:37]: 🙈

**Review by literat — APPROVED** [review]
Nice ❤️

File: packages/web-react/src/components/Alert/README.md [hunk 3]
```diff
@@ -15,15 +15,30 @@ import { Alert } from '@lmc-eu/spirit-web-react/components';
...
-| `color`        | [Emotion Color dictionary][dictionary-color] | `success`        | no       | Color of the component     |
-| `iconName`     | `string`                                     | Based on `color` | no       | Icon used in Alert         |
-| `isCentered` . | `boolean`                                    | `false`          | no       | If true, Alert is centered |
+| Prop name      | Type                                         | Default   | Required | Description                |
+| -------------- | -------------------------------------------- | --------- | -------- | -------------------------- |
+| `children`     | `ReactNode`                                  | -         | yes      | Content of the Alert       |
+| `color`        | [Emotion Color dictionary][dictionary-color] | `success` | no       | Color of the component     |
+| `iconName`     | `string`                                     | `info` \* | no       | Icon used in Alert         |
+| `isCentered` . | `boolean`                                    | `false`   | no       | If true, Alert is centered |
```
[the remark this answers is not in the document]
Comment (crishpeen) [reply packages/web-react/src/components/Alert/README.md]: Fixed

**Pull request merged**